### PR TITLE
Language/php

### PR DESCRIPTION
**Pull request by JanSuthacheeva**
Add the 3 endpoints for php with the laravel framework

### CODE REVIEW

File: php/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
```
Comment (SethY8s) [inline php/.editorconfig]: Not sure if any of these files are like node_modules or a package.lock file, but anything like that does not need to be included. I actually need to update the node and next example to remove some of the bloat.

File: php/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Getting Started
+
+### Prerequisites
+
+Ensure you have PHP and composer and  installed on your machine. Furthermore it is also recommended to have NPM and Node.js installed.
+
+If not, follow the instructions for installing PHP regading the [PHP website](https://www.php.net/manual/en/install.php).
+Since the documentation on the official website can be a little bit overwhelming, especially for Windows,
+feel free to watch a [youtube video](https://www.youtube.com/watch?v=MPRLUd8Pmyo) which guides you through the PHP installation.
```
Comment (SethY8s) [inline php/README.md:13]: perfect! I really like the readme, super clear. I will test it on my local machine this weekend.

File: php/app/Http/Middleware/TrustProxies.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+class TrustProxies extends Middleware
+{
+    /**
+     * The trusted proxies for this application.
+     *
+     * @var array<int, string>|string|null
+     */
+    protected $proxies;
+
+    /**
```
Comment (SethY8s) [inline php/app/Http/Middleware/TrustProxies.php:17]: Is this pretty much setting up cors? This is not an issue, but just curious.

File: php/routes/web.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+use App\Http\Controllers\CoinController;
```
Comment (SethY8s) [inline php/routes/web.php:3]: structure looks great

File: php/.editorconfig (hunk 1 above)
Comment (JanSuthacheeva) [reply php/.editorconfig]: This is similar to prettier - just some style configs. I added it to gitignore

**Comment by JanSuthacheeva** [comment]
I deleted most of the stuff that is unnecessary. Most of the staff thats in the project now is somehow necessary to run the applicaton. I think you can delete more stuff if you dig deeper into the config files and edit them but that is beyond my knowledge as of today.

File: php/app/Http/Middleware/TrustProxies.php (hunk 3 above)
Comment (JanSuthacheeva) [reply php/app/Http/Middleware/TrustProxies.php:17]: This is some laravel stuff that comes out of the box. tbh its beyond my knowledge as of today.

**Comment by SethY8s** [comment]
Sounds great! Thank you so much for contributing!

**Pull request merged**